### DISCUSSION TITLE
UGS and Fusion.... cant post process..

### DISCUSSION THREAD

**Discussion by ChunkiieCustomz** [winder/Universal-G-Code-Sender#2609] [Help]
So im new but I followed the tutorials verbatim. I can get it to be added but then when i try to add post to ugs it requests like the video says... then cant find config file..
![Screenshot 2024-09-13 193119](https://github.com/user-attachments/assets/353cf787-9be7-49b3-a128-b2872afcfc6a)

**Comment by breiler** [comment]
I don't use Fusion 360 myself so I have no clue. Maybe check with the developers that made the post processor?
https://github.com/tapnair/UGS_Fusion
